src/ec/packages/ec-component-description-list/package.json

```diff
@@ -2,14 +2,14 @@
   "name": "@ecl-twig/ec-component-description-list",
   "author": "European Commission",
   "license": "EUPL-1.1",
-  "version": "2.9.1",
+  "version": "2.11.0",
   "description": "ECL Twig - EC Description list",
   "publishConfig": {
     "access": "public"
   },
   "devDependencies": {
     "@ecl/ec-component-list": "2.11.0",
```

Comment:
it should refers to component `ec-component-description-list` (version 2.11.0).
component `ec-component-list` is deprecated